src/test/java/uk/gov/dhsc/htbhf/steps/Constants.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.dhsc.htbhf.steps;
+
+public class Constants {
+
+    public static final String DAY = "30";
```

Comment:
Could we rename these to have a `DOB_` prefix?

Reply:
Changed.